packages/teams-js/src/public/media.ts

```diff
@@ -414,15 +416,16 @@ export namespace media {
      */
     protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
-      const err = isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
-      if (err) {
+      try {
+        throwExceptionIfApiIsNotSupported(nonFullScreenVideoModeAPISupportVersion);
+      } catch (err) {
         if (callback) {
-          callback(err);
+          return callback(err);
         }
-        return;
       }
 
       const params: MediaControllerParam = { mediaType: this.getMediaType(), mediaControllerEvent: mediaEvent };
+
       sendMessageToParent('media.controller', [params], (err?: SdkError) => {
         if (callback) {
           callback(err);
```

Comment:
Do we need to throw here if callback is undefined? I assume if we don't the Promise version of the function won't fail correctly in this case. If we do need to make that change we should add corresponding unit tests